Replace geek comic

ccronje opened the pull request:
See https://github.com/LibraryCarpentry/lc-data-intro/issues/38

jezcope commented:
Thanks @ccronje, looks good to me.

ostephens commented:
Ah - I see https://github.com/LibraryCarpentry/lc-data-intro/commit/83cf76f5c86f49bb9e779717d4ac42c559447810#diff-3b96395c2a509c7af6cbe447d9ed867f reverted #43 - but I'm not clear why this happened?

ccronje commented:
That was my revert @ostephens - the fix didn't render but I also realised we hadn't addressed the issues i.e. the new comic resolved 'winner/loser' but not the term 'geek'.
